.github/workflows/ci.yaml

```diff
@@ -133,18 +134,12 @@ jobs:
     steps:
       - name: ⤵️ Check out code from GitHub
         uses: actions/checkout@v3
-      - name: 🏗 Set up build cache
```

Comment:
Not sure if we should change here from what Frenck is doing, but I am not an expert in Github Actions, so I trust you that this is a better solution to change the caching.